app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,49 @@
   </head>
 
   <body>
+  <nav class="navbar navbar-expand-lg navbar-light style="background-color: #e3f2fd;">
+
+    <h1><a class="navbar-brand" href="#">Obituary Hub</a></h1>
```

Comment:
`<h1>` didn't make it super huge as expected. I think we will need to put some work into making all the text across the site extra large, or better yet adding a giant obvious button that makes all the text gigantic if that is the user's preference

Reply:
good idea, that will contribute towards site accessibility with the target audience

Reply:
Curious if bootstrap has any features we can implement for this issue.